sphinx/source/docs/user_guide/styling.rst

```diff
@@ -20,6 +20,29 @@ following ways:
 
 .. _userguide_styling_visual_properties:
```

Comment:
I think the ref labels are swapped (this one with the one below in the next section)